packages/store-sync/src/query-cache/common.ts

```diff
@@ -41,13 +41,13 @@ type queryConditions<tables extends Tables> = {
   [tableName in keyof tables]: tableConditions<tableName & string, tables[tableName]>;
 }[keyof tables];
 
+export type QuerySubjects<tables extends Tables = Tables> = {
+  readonly [k in keyof tables]?: readonly [keyof tables[k]["schema"], ...(keyof tables[k]["schema"])[]];
+};
```

Comment:
not sure if this is any more readable:
```suggestion
export type QuerySubject<table extends Table = Table> = readonly [keyof table["schema"], ...(keyof table["schema"])[]];

export type QuerySubjects<tables extends Tables = Tables> = {
  readonly [k in keyof tables]?: QuerySubject<tables[k]>;
};
```

Reply:
done!